File: Scripts/JavaScript/sum.js
function handle(g) {
    const a = Number(g.Args?.a ?? 0);
    const b = Number(g.Args?.b ?? 0);
    if (typeof logInfo === "function") logInfo(`Somando ${a} + ${b}`);
    return { ok: true, sum: a + b, at: (g.NowUtc || new Date()).toString() };
}

File: Scripts/Python/sum.py
import datetime

def handle(g):
    args = g.get("Args", {}) or {}
    a = float(args.get("a", 0))
    b = float(args.get("b", 0))
    return {
        "ok": True,
        "sum": a + b,
        "at": datetime.datetime.utcnow().isoformat() + "Z"
    }
